spec/request/disability_compensation_form_request_spec.rb

```diff
@@ -157,7 +141,7 @@
 
     context 'with invalid json body' do
       it 'returns a 422' do
-        post '/v0/disability_compensation_form/submit', params: { 'form526' => nil }.to_json, headers: headers
+        post '/v0/disability_compensation_form/submit_all_claim', params: { 'form526' => nil }.to_json, headers: headers
```

Comment:
Were there not specs covering the ‘/v0/disability_compensation_form/submit_all_claim’ endpoint to test 422? I see you’re replacing /submit with /submit_all_claim but I would expect a spec to just be deleted

Reply:
there wasn't a spec for submit_all_claim - 422 before. There should've but there was't.  These two endpoints share a ton of code because essentailly `submit_all_claim` does everything `submit` does and more... but perhaps with a slightly different json strucutre.

Reply:
Ok great, thanks for the reply! Looks good